src/VuetifyFileManager.vue

```diff
@@ -88,28 +86,32 @@ export default {
     },
     methods: {
         folderSelected(selectedFolderId) {
-            this.getFolderContent(selectedFolderId).then((content) => {
+            this.selectedFolderId = selectedFolderId;
+
+            this.getFolderContent(this.selectedFolderId).then((content) => {
                 this.files = content;
             });
         },
         callGetFolders() {
             this.getFolders().then(folders => {
                 this.folders = folders;
-                this.folderSelected(folders[0].id);
+                this.selectedFolderId = this.selectedFolderId === null ? folders[0].id : this.selectedFolderId;
```

Comment:
Not sure if you're still updating this.

But it's better to store the whole reference to selected folder instead of just the id. If you need another property of a `folder` object, you have to work your way around again.